ionic-app/components/pages/Tabs.jsx

```diff
@@ -0,0 +1,39 @@
+import { Redirect, Route } from 'react-router-dom';
+import { IonRouterOutlet, IonTabs, IonTabBar, IonTabButton, IonIcon, IonLabel } from '@ionic/react';
+import { IonReactRouter } from '@ionic/react-router';
+import { cog, flash, list } from 'ionicons/icons';
+
+import Home from './Feed';
+import Lists from './Lists';
+import ListDetail from './ListDetail';
+import Settings from './Settings';
```

Comment:
Codacy has a fix for the issue: [Strings must use doublequote.](https://app.codacy.com/gh/toost-net/toost-cloud/pullRequest?prid=8079923)
```suggestion
import Settings from "./Settings";
```